README.md

```diff
@@ -11,7 +11,7 @@ helpfiles in the `doc/` directory. The helpfiles are also available via
 # Supported File-types
 
 * [Bazel](https://www.github.com/bazelbuild/bazel) BUILD files (buildifier)
-* C, C++ (clang-format)
+* C, C++, Cuda (clang-format)
```

Comment:
I don't think Cuda needs to be distinct from C/C++ here.

Reply:
Meaning just revert this change, and "C, C++" implies Cuda as well?

Reply:
If it does need to be explicitly listed, note there's also a listing at https://github.com/google/vim-codefmt/blob/1b76987b6719bee259ba8d1e03a1e7f624a3615f/plugin/register.vim#LL30C20-L30C20 you haven't updated, and generated docs based on that list.

Reply:
> "C, C++" implies Cuda as well?

Yeah, I think CUDA just adds a few extensions to C/C++.

Reply:
K, so @Presburger if so you could just revert this and keep just the vimscript change and I can merge it.

Reply:
> K, so @Presburger if so you could just revert this and keep just the vimscript change and I can merge it.

OK, well, Cuda is a extend of C++ & C, i agree with yours suggest.